oxidation/libparsec/crates/serialization_format/src/parser/protocol.rs

```diff
@@ -342,7 +342,7 @@ impl Cmds {
                     }
 
                     pub fn load(buf: &[u8]) -> Result<Self, &'static str> {
-                        ::rmp_serde::from_slice(buf).map_err(|_| "Deserialization failed")
+                        ::rmp_serde::from_slice(buf).map_err(|_| {"Deserialization failed"})
```

Comment:
```suggestion
                        ::rmp_serde::from_slice(buf).map_err(|_| "Deserialization failed")
```
Braces not required here